controllers/terminal/scripts/Dockerfile-terminal-all

```diff
@@ -0,0 +1,71 @@
+FROM centos:7.6.1810
+# install tty
+RUN yum install -y wget &&  yum install -y bind-utils && \
+    wget https://github.com/tsl0922/ttyd/releases/download/1.4.2/ttyd_linux.x86_64 && \
+    chmod +x ttyd_linux.x86_64 && \
+    mv ttyd_linux.x86_64 /usr/bin/ttyd
+
+# install golang
+RUN cd /root && wget https://dl.google.com/go/go1.12.linux-amd64.tar.gz && \
+	tar zxvf go1.12.linux-amd64.tar.gz && \
+	rm -rf go1.12.linux-amd64.tar.gz && mkdir /root/work
+ENV GOROOT=/root/go GOPATH=/root/work PATH=$PATH:/root/go/bin:/root/work/bin
+
+# ENV APISERVER="https://127.0.0.1:6443"
+# ENV USER_TOKEN="xxx"
+# ENV NAMESPACE="default"
+# ENV USER_NAME="fanux"
+# ENV TERMINAL_ID="UUID"
+
+# install kubectl
+RUN wget https://dl.k8s.io/v1.13.3/kubernetes-client-linux-amd64.tar.gz && \
```

Comment:
Is it okay to mount it directly with kubectl?

Reply:
@fanux   for all k8s version build the image ？ Isn't this a little heavy? At present, each node in Sealos's cluster has kubectl. Can we just mount it?

Reply:
Seems this file is direct copied/pasted from other repo and needs some update.

As `k8s`|`kubectl`|`containerd` command, I prefer `mount` instead of `install` to simpler k8s manage as when this application is applied, we can sure the host nodes already have these bins.

To better clarify, the all-in-one docker image just install needed softwares like `go`,`ruby`,`perl` etc..., and mount `kubectl` bins, to leave its version or arch to host.

Reply:
@cuisongliu different kubectl version use different terminal version, you can't make sure every node has kubectl.